src/decorators/index.ts

```diff
@@ -11,4 +11,3 @@ export { GraphQLResolver } from "./GraphQLResolver";
 export { Mutation } from "./Mutation";
 export { Query } from "./Query";
 export { Root } from "./Root";
-export { Subscription } from "./Subscription";
```

Comment:
`Subscription` decorator exist but it's not implemented yet. I would rather comment this line to prevent accidental usage of this decorator (intellisense stuff).